### PR TITLE
chore/Flux 0.172

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
new Flux 0.172 release 
https://github.com/influxdata/flux/releases/tag/v0.172.0

Autogenerated Docs.
Feel free to pull down the branch and view the Flux docs for these changes. 
The biggest changes are all of the new functions in the `join` package of stdlib

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/flux/v0.x/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,30 @@ aliases:
...
+## v0.172.0 [2022-06-24]
+
+### Features
+- Add multiple new join functions to the `join` package such as `join.full()`.
+- Add `initialZero` parameter to derivative function. 
+- Allow features to enable builtin statements. 
+- Provide the comments for each `Symbol` from `PackageExports`.
+- Suggestions now start off by default and added a new flag.
+- Add builtin function time to the `date` package to convert any timeable into datetime. 
+- Allow vector types to be specified in flux source.
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:22]:
```suggestion
- Allow vector types to be specified in Flux source. 
```

File: content/flux/v0.x/release-notes.md [hunk 2]
```diff
@@ -10,6 +10,30 @@ aliases:
...
+- Add multiple new join functions to the `join` package such as `join.full()`.
+- Add `initialZero` parameter to derivative function. 
+- Allow features to enable builtin statements. 
+- Provide the comments for each `Symbol` from `PackageExports`.
+- Suggestions now start off by default and added a new flag.
+- Add builtin function time to the `date` package to convert any timeable into datetime. 
+- Allow vector types to be specified in flux source. 
+
+### Bug fixes
+- Replace extra boolean parameter for suggestions with repl options.
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:25]:
```suggestion
- Replace extra boolean parameter for suggestions with Flux REPL options. 
```

File: content/flux/v0.x/release-notes.md [hunk 3]
```diff
@@ -10,6 +10,30 @@ aliases:
...
+- Add `initialZero` parameter to derivative function. 
+- Allow features to enable builtin statements. 
+- Provide the comments for each `Symbol` from `PackageExports`.
+- Suggestions now start off by default and added a new flag.
+- Add builtin function time to the `date` package to convert any timeable into datetime. 
+- Allow vector types to be specified in flux source. 
+
+### Bug fixes
+- Replace extra boolean parameter for suggestions with repl options. 
+- Remove `testing.load()` from yield test.
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:26]: I think yield test may refer to [testutil.yield()](https://docs.influxdata.com/flux/v0.x/stdlib/internal/testutil/yield/), but lets double-check

File: content/flux/v0.x/release-notes.md [hunk 4]
```diff
@@ -10,6 +10,30 @@ aliases:
...
+- Allow features to enable builtin statements. 
+- Provide the comments for each `Symbol` from `PackageExports`.
+- Suggestions now start off by default and added a new flag.
+- Add builtin function time to the `date` package to convert any timeable into datetime. 
+- Allow vector types to be specified in flux source. 
+
+### Bug fixes
+- Replace extra boolean parameter for suggestions with repl options. 
+- Remove `testing.load()` from yield test. 
+- Fix a bug in how sort nodes created for new join.
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:27]:
```suggestion
- Fix a bug in how sort nodes are created for a new join. 
```

File: content/flux/v0.x/release-notes.md [hunk 5]
```diff
@@ -10,6 +10,30 @@ aliases:
...
+- Provide the comments for each `Symbol` from `PackageExports`.
+- Suggestions now start off by default and added a new flag.
+- Add builtin function time to the `date` package to convert any timeable into datetime. 
+- Allow vector types to be specified in flux source. 
+
+### Bug fixes
+- Replace extra boolean parameter for suggestions with repl options. 
+- Remove `testing.load()` from yield test. 
+- Fix a bug in how sort nodes created for new join. 
+- Removed extra indentation for testcases.
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:28]:
```suggestion
- Removed extra indentation for test cases. 
```

File: content/flux/v0.x/release-notes.md [hunk 6]
```diff
@@ -10,6 +10,30 @@ aliases:
...
+- Add builtin function time to the `date` package to convert any timeable into datetime. 
+- Allow vector types to be specified in flux source. 
+
+### Bug fixes
+- Replace extra boolean parameter for suggestions with repl options. 
+- Remove `testing.load()` from yield test. 
+- Fix a bug in how sort nodes created for new join. 
+- Removed extra indentation for testcases. 
+- Retain the package for identifier referencing the prelude. 
+- Only return error in tests if an assertion fails.
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:30]:
```suggestion
- Only return an error in tests if an assertion fails. 
```

**Review by kelseiv — APPROVED** [review]

File: content/flux/v0.x/release-notes.md (hunk 3 above)
Comment (sunbryely-influxdata) [reply content/flux/v0.x/release-notes.md:26]: thanks for catching this. i checked and it does refer to testutil().yield()

**Pull request merged**